README.md

```diff
@@ -32,17 +32,17 @@ Or you can use `StrimziKafkaCluster` for a Kafka cluster with multiple Kafka bro
 
 ## How to use the Test container?
 
-Using a given test container takes three simple steps:
-1. Specify two environment variables
-2. Add the Strimzi test container as a Maven dependency
-3. Run Strimzi test container
+Using Strimzi test container takes two simple steps:
+1. Add the Strimzi test container as a Maven dependency
+2. Run Strimzi test container
     1. default configuration
     2. (optional) run Strimzi test container with additional configuration
 
-First, you need to specify two essential environment variables:
+You can specify environment variables to configure the container image to run:
 
-- `STRIMZI_TEST_CONTAINER_KAFKA_VERSION` - specification of the Kafka version to run with the given container. For example, `3.0.0`.
-- `STRIMZI_TEST_CONTAINER_IMAGE_VERSION` - specification of the Strimzi test container image version that was built in the `test-container-images` repositories. For example, `0.1.0`.
+- `STRIMZI_TEST_CONTAINER_KAFKA_VERSION` - the Kafka version to run with the given container. Default : `3.0.0`.
```

Comment:
We are not gonna have these ENVs about (test-container, Kafka version). In this PR https://github.com/strimzi/test-container/pull/7 I have replaced it with a mechanism that will fetch these properties and use it.